lib/project_types/script/layers/application/create_script.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+require "shopify_cli"
+
+module Script
+  module Layers
```

Comment:
Why do we need `Layers`?

Reply:
It's not necessary. I just liked having some separation between our ddd layers and command layer code, like the script module in our fork